# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webaudio/the-audio-api/the-oscillatornode-interface/osc-basic-waveform.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/webaudio/the-audio-api/the-oscillatornode-interface/osc-basic-waveform.html.ini
--- a/testing/web-platform/meta/webaudio/the-audio-api/the-oscillatornode-interface/osc-basic-waveform.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/webaudio/the-audio-api/the-oscillatornode-interface/osc-basic-waveform.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -41,12 +41,12 @@
   [X Sine: SNR (db) is not greater than or equal to 130.95. Got 112.67669857220825.]
     expected: FAIL
 
-  [X Custom: SNR (db) is not greater than or equal to 122.43. Got 112.76579764697749.]
-    expected: FAIL
-
   [X Custom: SNR (db) is not greater than or equal to 138.76. Got 132.76794522938812.]
     expected: FAIL
 
   [X Custom: 100 Hz does not equal [1,1.0141456127166748,1.0280853509902954,1.0418163537979126,1.0553359985351562,1.0686413049697876,1.081729769706726,1.094598650932312,1.1072453260421753,1.1196671724319458,1.1318618059158325,1.1438266038894653,1.1555593013763428,1.1670573949813843,1.1783186197280884,1.189340591430664...\] with an element-wise tolerance of {"absoluteThreshold":0.0000018478,"relativeThreshold":0}.\n\tIndex\tActual\t\t\tExpected\t\tAbsError\t\tRelError\t\tTest threshold\n\t[132\]\t6.4742147922515869e-1\t6.4741957187652588e-1\t1.9073486328125000e-6\t2.9460781163660346e-6\t1.8477999999999999e-6\n\t[133\]\t6.2944263219833374e-1\t6.2944072484970093e-1\t1.9073486328125000e-6\t3.0302275615673585e-6\t1.8477999999999999e-6\n\t[135\]\t5.9310543537139893e-1\t5.9310334920883179e-1\t2.0861625671386719e-6\t3.5173677065245060e-6\t1.8477999999999999e-6\n\t[136\]\t5.7475423812866211e-1\t5.7475227117538452e-1\t1.9669532775878906e-6\t3.4222627316729277e-6\t1.8477999999999999e-6\n\t[137\]\t5.5628657341003418e-1\t5.5628448724746704e-1\t2.0861625671386719e-6\t3.7501721061127269e-6\t1.8477999999999999e-6\n\t...and 97 more errors.\n\tMax AbsError of 4.5299530029296875e-6 at index of 197.\n\t[197\]\t-6.1586797237396240e-1\t-6.1587250232696533e-1\t4.5299530029296875e-6\t7.3553421947141031e-6\t1.8477999999999999e-6\n\tMax RelError of 4.6918106549645650e-4 at index of 165.\n\t[165\]\t7.5594326481223106e-3\t7.5558875687420368e-3\t3.5450793802738190e-6\t4.6918106549645650e-4\t1.8477999999999999e-6\n]
     expected: FAIL
 
+  [X Custom: SNR (db) is not greater than or equal to 122.43. Got 112.76579764697749.]
+    expected: FAIL
+